package.json

```diff
@@ -6,6 +6,10 @@
   ],
   "scripts": {
     "build": "lerna run build",
+    "build:visual": "docker build -t screenshot:latest ./;",
+    "test:visual": "yarn build:visual && docker run -t screenshot:latest yarn test:integration",
+    "copy:test:visual": "echo 'do something to get __diff_output__ out of the container'",
+    "test:visual:execute": "cd packages/swarm-components/ && yarn test || true && ls src/__image_snapshots__",
```

Comment:
Not sure what this does - seems like it runs it locally do we need this?

Reply:
I believe certain steps like copying and executing are now `docker run` commands instead of within the docker file, in which case, some of these scripts might not be needed anymore. However, we'll need the instructions for `docker run`.

Reply:
no longer needed/used